simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,33 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    if len(lst) == 0:
```

Comment:
If you plan to call “len(lst)” many times, why don’t move and use lenLst = len(lst)?